### PR TITLE
Commit of assignment_five

### PR DESCRIPTION
**Pull request by klickyfan**
I am sorry this is late. Thanks for your patience!

### CODE REVIEW

**Comment by klickyfan** [comment]
I assume it is ok to have failing database-dependent tests? All my tests pass locally.

**Comment by smarks** [comment]
fix the circle ci stuff for next time.

File: src/main/java/edu/kimjones/advancedjava/stock/StockQuoteApplication.java [hunk 1]
```diff
@@ -76,38 +80,62 @@ private void doMain(String args[]) throws IOException {
...
+                System.out.printf("The price of stock %s at time %tD %tR is %.2f %n", temp.getStockSymbol(), temp.getDateRecorded(), temp.getDateRecorded(), temp.getStockPrice());
+            }
 
-        for (StockQuote temp : hourlyQuoteList) {
-            System.out.printf("The price of stock %s at time %tD %tR is %.2f %n", temp.getStockSymbol(), temp.getDateRecorded(), temp.getDateRecorded(), temp.getStockPrice());
         }
+        catch (StockServiceException exception) {
+            System.out.printf("An exception was caught: " + exception.getMessage());
+        }
+
```
Comment (smarks) [inline src/main/java/edu/kimjones/advancedjava/stock/StockQuoteApplication.java:139]: Nice clean up here, but maybe make it even cleaner by breaking the long flow of code into discreet methods.

**Review by smarks — APPROVED** [review]
Another good job .

**Pull request merged**